packages/web-api-scan-runner/src/runner/runner.ts

```diff
@@ -84,7 +89,20 @@ export class Runner {
         }
 
         this.logger.logInfo(`Writing page scan run result to a storage.`);
-        await this.onDemandPageScanRunResultProvider.updateScanRun(pageScanResult);
+        const fullPageScanResult = await this.onDemandPageScanRunResultProvider.updateScanRun(pageScanResult);
+
+        const featureFlags = await this.getDefaultFeatureFlags();
+        this.logger.logInfo(`sendNotification feature flag ${featureFlags.sendNotification}`);
+        if (featureFlags.sendNotification && !this.scanNotifyUrlEmpty(fullPageScanResult.notification)) {
+            this.logger.logInfo(`Sending notification to ${fullPageScanResult.notification.scanNotifyUrl}`);
+            await this.notificationDispatcher.dispatchOnDemandScanRequests(
```

Comment:
rename dispatchOnDemandScanRequests. it is not dispatching scan requests.
may be rename to sendScanCompletedNotification?

Reply:
or rename class to NotificationQueueMessageSender & the method name to send.